### ISSUE TITLE
`LogisticSaturation` class does not document all of it's parameters

### ISSUE THREAD

**Issue by cluhmann** [pymc-labs/pymc-marketing#1188]
The logistic saturation function seems to come in 2 varieties. There is the `LogisticSaturation` class defined [here](https://github.com/pymc-labs/pymc-marketing/blob/e80ea88f4f8f4ba3f33641f13f4f350f679656d1/pymc_marketing/mmm/components/saturation.py#L178C7-L178C25) and then there is the `logistic_saturation` function defined [here](https://github.com/pymc-labs/pymc-marketing/blob/e80ea88f4f8f4ba3f33641f13f4f350f679656d1/pymc_marketing/mmm/transformers.py#L451).  The former has no documentation of parameters (though there is a brief vignette?) and instead says it is a "Wrapper around logistic saturation function" and see the documentation on the latter.  The problem is that there the former takes a parameter (`beta`) that is not found anywhere in the latter.  And the vignette doesn't help because it passes in no parameter values.  At best, this seems very confusing.

Are there other functions and wrappers that behave like this?

**Comment by cluhmann** [comment]
Looking at it again, I realize that the code in the docstring isn't a vignette, but rather generates a plot.  I don't think we need this in the `LogisticSaturation` class docstring as it a) doesn't show off the invoking code (and thus is totally mysterious) and b) is redundant with the (much prettier and more informative) plots in the `logistic_saturation` function doc string. If the plot showed off plots for various values of `beta` and `lam` that would be unique and potentially useful, but right now it's not.

**Comment by wd60622** [comment]
All of the Saturation transformations get a scaling parameter `beta` in order to match with the Menten `alpha` parameter. It might be documented in the module since it applies to most of them. 

If you have a suggestion of how to improve it, please let me know.

**Comment by cluhmann** [comment]
So if there's a note added to the module, where would that show up?

**Comment by wd60622** [comment]
> So if there's a note added to the module, where would that show up?

That was part of my edit and realization; it is not there. I added to my action items

**Comment by cluhmann** [comment]
> > So if there's a note added to the module, where would that show up?
> 
> That was part of my edit and realization; it is not there. I added to my action items

Yeah, I was asking where it would show up **if it was added**.  I just don't have a sense of where I would find it in the docs if this issue was addressed in the way you are suggesting.

**Comment by wd60622** [comment]
I was thinking about the module docstring. So in `pymc_marketing.mmm.components.saturation.py` at the top. Do you have any other suggestions?

**Comment by cluhmann** [comment]
Sure, and where would it appear in the docs if it was there?  [Here](https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.components.saturation.html)?

**Comment by wd60622** [comment]
> Sure, and where would it appear in the docs if it was there? [Here](https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.components.saturation.html)?

Yes. Somewhere in that top section. Sound reasonable home for it?

**Comment by cluhmann** [comment]
I guess, but if people find their way to one of the constituent saturation components (looking for details about the [LogisticSaturation](https://www.pymc-marketing.io/en/stable/api/generated/pymc_marketing.mmm.components.saturation.LogisticSaturation.html)), then the use of `beta` will still not be evident. So how do we alert people to the actual set of parameters of the individual saturation components?

**Comment by cluhmann** [comment]
Yup, I think that works then.

**Comment by anevolbap** [comment]
PR up at #2586.
